src/gerbil/runtime/build.scm

```diff
@@ -60,4 +62,5 @@
   (displayln "building gerbil/runtime in " libdir)
   (set! *libdir* libdir)
   (update-gx-version)
-  (for-each compile *gx-modules*))
+  (let ((gscs (map start-gsc *gx-modules*)))   ;; Parallel compilation
```

Comment:
that's probably ok, because it's only 4 files, but we should still obey `GERBIL_BUILD_CORES` I think.